testdata/go/v4-alpha/memcached-operator/test/e2e/e2e_test.go

```diff
@@ -193,4 +224,120 @@ var _ = Describe("memcached", func() {
 			Eventually(getStatus, time.Minute, time.Second).Should(Succeed())
 		})
 	})
+
+	Context("Memcached Operator metrics", func() {
+		var portForwardCmd *exec.Cmd
```

Comment:
I am personally not a fan of using this port forwarding method. I think it puts a potential failure point in the test that is dependent on the environment the test is being run in. If the test happens to land on using a port that is already in use then the test will fail. It also seems like there is the potential that if killing the process fails then the test will fail and we could be potentially orphaning the process that is running the port forwarding command. 

Ideally, we don't have tests that rely on a variables such as a local machine port being available.

I would much prefer to see that a pod is created on the test cluster to curl the metrics similar to how it is done in our E2E tests here:
https://github.com/operator-framework/operator-sdk/blob/76ac4ba5d7d77590ee0e973c5b97595fcf447495/test/e2e/go/cluster_test.go#L122-L174

Or similar to how it is done in the Kubebuilder go/v3 E2E tests: 
- https://github.com/kubernetes-sigs/kubebuilder/blob/d67473979371c60b213742a9a7f19cf67b6bab43/test/e2e/v3/plugin_cluster_test.go#L193
- https://github.com/kubernetes-sigs/kubebuilder/blob/d67473979371c60b213742a9a7f19cf67b6bab43/test/e2e/v3/plugin_cluster_test.go#L299-L385